### ISSUE TITLE
Remove draw tool when in Static Mode

### ISSUE THREAD

**Issue by antobenve** [unep-grid/mapx#935]
In static mode there is no way to handle the geometries created via the draw tool.
Having the possibility to draw geometries and save them can confuse the users if they won't find the layer anywhere afterwards.

I propose the tool is removed from the right-hand buttons when in static mode.

**Comment by fxi** [comment]
-> we could also add the GeoJSON's view to the legend box, with a download link ?

**Comment by antobenve** [comment]
Yes, it could be a good alternative if the actions required to do so by the user can be handled in the static mode.
